src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -1,8 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
```

Comment:
Now what happens if you build this csproj locally without having this default? I suspect it fails. We should see about keeping these defaults for local project builds without have to specify the Configuration directly at the command line.